examples/tts/conf/es/fastpitch_align_44100_ipa_multi.yaml

```diff
@@ -13,10 +13,8 @@ sup_data_types: ["align_prior_matrix", "pitch", "speaker_id"]
 pitch_fmin: 65.40639132514966
 pitch_fmax: 2093.004522404789
 
-# these frame-wise values depend on pitch_fmin and pitch_fmax, you can get values
-# by running `scripts/dataset_processing/tts/extract_sup_data.py`
-pitch_mean: ???  # e.g. 178.86880493164062 for 174 speakers in https://research.google/pubs/pub49150/
```

Comment:
Is this backward compatible? Meaning, with these changes can the checkpoints published in NGC be loaded and finetuned/inferred on?

Reply:
Yes. To confirm, I will test it with one of the existing English FastPitch models on NGC.

Reply:
Great.